posthog/client.py

```diff
@@ -366,11 +366,8 @@ def _load_feature_flags(self):
 
         except APIError as e:
             if e.status == 401:
-                raise APIError(
```

Comment:
how do you feel about raising in debug mode?

I don't want to be completely silent all the time, that sucks devX wise :/